include/grpc/impl/codegen/port_platform.h

```diff
@@ -229,7 +230,6 @@
 #define GPR_POSIX_SYNC 1
 #define GPR_POSIX_TIME 1
 #define GPR_GETPID_IN_UNISTD_H 1
-/* TODO(mxyan): Remove when CFStream becomes default */
 #ifndef GRPC_CFSTREAM
```

Comment:
Why TODO is not done but removed?

Reply:
We can only turn CFStream on by default on iOS; the following 3 lines are for non-iOS platforms; they still need this change and we probably will not make CFStream default on them.